lib/presenter/subject_add.dart

```diff
@@ -77,22 +78,31 @@ class SubjectAddState extends State<SubjectAdd> {
       setState(() {
         isValidated = true;
       });
+      SnackBar snackBar = SnackBar(
+        content: Text(
+          "all inputs are correct!.",
+          style: TextStyle(color: Colors.white),
+        ),
+        duration: Duration(seconds: 2),
+        backgroundColor: Colors.green,
+      );
+      Scaffold.of(context).showSnackBar(snackBar);
+      return;
     } else {
-      showDialog<bool>(
-        context: context,
-        builder: (BuildContext context) {
-          return AlertDialog(
-              title: const Text('Please check your input again'),
-              actions: <Widget>[
-                FlatButton(
-                  child: const Text('Okay'),
-                  onPressed: () {
-                    Navigator.of(context).pop(true);
-                  },
-                )
-              ]);
-        },
+      SnackBar snackBar = SnackBar(
+        content: Text(
+          "please check your input again.",
```

Comment:
Snackbars should start with a capital letter.

Reply:
fixed it.